src/components/account-header/account-header.yaml

```diff
@@ -102,10 +102,10 @@ examples:
           - focus: ".hmrc-account-menu__link--home"
       account-menu-messages-hovered:
         beforeTakingScreenshot:
-          - hover: ".hmrc-account-menu__link[href='/messages/href']"
+          - hover: ".hmrc-account-menu__link[href$='/messages/href']"
```

Comment:
when we create the static site, it changes these urls to have domain in them - so swapped to match on suffix of href to make them still match afterwards